docs/docs/llms/llm-setup.mdx

```diff
@@ -47,8 +47,17 @@ interaction between the two.
 
 To configure the API token, follow these steps:
 
-1. [Instructions to get your API token]
-2. [Instructions to set the token in your LLM configuration]
+1. If you haven't already, sign up for an account on the OpenAI platform.
+   Navigate to the OpenAI Key Management page at https://platform.openai.com/account/api-keys.
+   Once there, click on the "Create New Secret Key" button to initiate the process of
+   obtaining your API key.
+
+2. Setting the API key as an environment variable, you can use the following command in a
+   terminal or command prompt:
+   ```shell Copy code
```

Comment:
we might want to also give instructions on how to do it on Powershell. To that end we could use tabs like [here](https://github.com/RasaHQ/rasa/blob/f9682d4a43f3aaa2bfb9c4729588371030c21886/docs/docs/deploy/deploy-rasa.mdx#L29-L50)